Makefile

```diff
@@ -26,9 +26,15 @@ $(ENV_FILE):
 
 $(IO_DIR)/%.sqlite: $(ENV_FILE)
 	docker run --rm -e DB_DIR=/io -v$(IO_DIR):/io --env-file $(ENV_FILE) --entrypoint python3 $(IMAGE) /server/tests/gen_sqlite_db.py
-	@touch $@
+	if [ ! -f $@ ]; then @touch $@; fi
```

Comment:
This avoids an error thrown when the database already exists. It happens because the data is created from the inside of the container with root privileges, so it's not possible to `touch` it without sudoing.

Reply:
Does it run every time now? We should probably `docker --user $$(id -u)` instead

Reply:
After reviewing this... what's the purpose of the `touch` here?
Aren't `mdb.sqlite` and `sdb.sqlite` fixtures generated by the `docker run ... gen_sqlite_db.py` above? If so, why is it needed to `touch` them?

Reply:
Otherwise `make` thinks that it needs to re-run the rule: the modification timestamps become out of order.

Reply:
to run again `$(IO_DIR)/%.sqlite`?
Why? if the passed file already exists, it is not needed to rerun the target, is it?
I locally tried after deleting the `touch`, and it didn't run the `docker run ... gen_sqlite_db.py` again after creating `mdb.sqlite` and `sdb.sqlite` fixtures for the first time.

Reply:
The databases used to be recreated over and over for me without `touch` - this is why I had to add it, apparently. Let me test.

Reply:
Yep, this is no longer needed now. Good! Then can you please remove this line completely.